code/game/jobs/access.dm

```diff
@@ -251,19 +252,19 @@ proc/get_all_job_icons()
 				"Squad Smartgunner",
 				"Squad Marine",
 				"Chief Medical Officer",
-				"Doctor",
+				"Medical Officer",
 				"Researcher",
-				"Military Police",
-				"Chief MP"
+				"Master at Arms",
+				"Command Master at Arms"
 				)
 
 /proc/get_paygrades(paygrade, size, gender)
```

Comment:
replace this with a proc on the skill/mind datum with a var on the skill datum with the short/long paygrade so you can just directly access the paygrade without this ugly switch that breaks everytime jobnames do